CHANGELOG.md

```diff
@@ -423,7 +423,7 @@ Removes Emergency-Alerts-related code
 
 ## 73.2.0
 
-* Adds a `include_notify_tag` parameter to `LetterPrintTemplate` so that bilingual letters can disable the NOTIFY tag on the English pages of a letter.
+* Adds a `includes_first_page` parameter to `LetterPrintTemplate` so that bilingual letters can disable the NOTIFY tag on the English pages of a letter.
```

Comment:
dont think we should be changing old changelogs? 🤔

Reply:
oops, good spot!